plugins/TimingTriggerCandidateMaker.cpp

```diff
@@ -1,55 +1,144 @@
 
-#include "trigger/TriggerCandidate.hh"
-#include "trigger/TriggerCandidateMaker.hpp"
+#include "TimingTriggerCandidateMaker.hpp"
 
-using clock = std::chrono::duration<double, std::ratio<1, 50'000'000>>;
+#include <chrono>
+
+using internal_clock = std::chrono::duration<double, std::ratio<1, 50'000'000>>;
```

Comment:
In ProtoDUNE-II and DUNE FD, the clock will be 62.5 MHz. Since we probably also need to support "old" ProtoDUNE-I data in the meantime, the clock frequency will have to become a configuration parameter